### ISSUE TITLE
Currently able to compile?

### ISSUE THREAD

**Issue by noperator** [jief666/hdimount#2]
@jief666, are you currently able to get the `wip` branch to compile? I'm assuming `fuse.cpp` is the main executable. I'm attempting to compile on Linux x86_64 and am running into some dependency issues that make me wonder if it was intended for compilation on macOS, which doesn't make sense to me since the tool's purpose is to open encrypted sparsebundle files on Linux.

**Comment by jief666** [comment]
I mostly compile on Os X, but yes, the target is Linux. Running on Os X allows me to mount disk images trough Os X at the same time and compare. I found quite a few bugs in Darling-dmg that way.
It's perfectly possible that this branch doesn't compile. It's my work in progress branch and I focused more on bugs than cross compilation lately.
I'm away with just a little laptop until the end of the month. I'll fix this then.
You are welcome to post here and I'll try to help.
